### PR TITLE
Srikanth grp by fix

### PR DESCRIPTION
**Pull request by srikanthgaddam2**

### CODE REVIEW

File: data_validation/data_validation.py [hunk 1]
```diff
@@ -300,7 +301,14 @@ def _execute_validation(self, validation_builder, process_in_memory=True):
         if process_in_memory:
             source_df = self.config_manager.source_client.execute(source_query)
             target_df = self.config_manager.target_client.execute(target_query)
-
+            #Loop through the columns to find timestamp and convert the data type to timestamp with no tz
+            for column in target_df.columns:
+                if is_datetime64_ns_dtype(target_df[column]):
+                    print(target_df[column])
```
Comment (simson345) [inline data_validation/data_validation.py]: Do we still need the print statements?

File: data_validation/data_validation.py [hunk 2]
```diff
@@ -301,6 +301,11 @@ def _execute_validation(self, validation_builder, process_in_memory=True):
         if process_in_memory:
             source_df = self.config_manager.source_client.execute(source_query)
             target_df = self.config_manager.target_client.execute(target_query)
+            #Replacing the /n and /r for all columns - Srikanth
+            source_df = source_df.replace(r'\n','', regex=True)
```
Comment (simson345) [inline data_validation/data_validation.py]: Can we try merging these four into 2 regex statements using the "|" in regex? 
target_df = target_df.replace(r'\n|\r','', regex=True)

**Review by simson345 — CHANGES_REQUESTED** [review]

File: data_validation/data_validation.py (hunk 1 above)
Comment (srikanthgaddam2) [reply data_validation/data_validation.py]: agreed

File: data_validation/data_validation.py (hunk 2 above)
Comment (srikanthgaddam2) [reply data_validation/data_validation.py]: Tried using source_df = source_df.replace(to_replace=[r"\\t|\\n|\\r", "\t|\n|\r"], value=["",""], regex=True, inplace=True) but it was accepting.. will try again

**Pull request merged**